src/uv-common.c

```diff
@@ -944,7 +944,7 @@ __attribute__((destructor))
 void uv_library_shutdown(void) {
   static int was_shutdown;
 
-  if (uv__load_relaxed(&was_shutdown))
+  if (uv__load_int_relaxed(&was_shutdown))
```

Comment:
@bnoordhuis Shouldn't this be a `atomic_exchange_explicit`? Otherwise there's a potential race between this check and when it's stored below.

Reply:
Yes, it should. I probably mindlessly changed it when I moved over all static variables from non-atomic to atomic loads/stores.